README.md

```diff
@@ -1 +1,79 @@
-# decaf-metabolica
\ No newline at end of file
+# Getting started
+
+Welcome to our cell factory design and analysis platform! We develop the platform as part of the [DD-DeCaF](http://dd-decaf.eu) project with the goal to put model-guided and data-driven design into practice in industrial biotechnology. The platform will enable the following key technologies:
+
+* Metagenomics-enabled design of novel enzymes and biochemical pathways.
+* Omics data-driven design of cell factories for the production of chemicals and proteins.
+* Analysis and design of microbial communities relevant to human health, industrial biotechnology and agriculture.
+
+With this interactive web application, we target primarily non-expert users that have a need to analyze omics data and compute strains designs. An advance programming interface is provided for expert users ([API docs](https://docs.dd-decaf.eu/)) enabling them deviate from the default workflows and perform custom analyses.
+
+The platform is currently in __beta__ and tested primarily with Google Chrome. If you'd like to stay up-to-date with new releases, please [subscribe](https://tinyletter.com/dd-decaf) to our quarterly newsletter. We will be eternally grateful for feedback in case you miss features or you encountered a problem (for now please drop us a message at [niso@biosustain.dtu.dk](mailto:niso@biosustain.dtu.dk)).
+
+The following sections provide an overview of what you can currently do on platform. Have fun!
+
+### Login (optional)
+
+You can log in [here](https://app.dd-decaf.eu/login) or by navigating to
```

Comment:
Suggest we use relative links so you're not e.g. unknowingly sent from staging to production

Reply:
Wouldn't that break the links if people would look at the markdown file on GitHub? Ideally, no one would ever look at staging besides us.